apps/src/eval/eval.js

```diff
@@ -106,6 +106,27 @@ Eval.init = function(config) {
       var background = document.getElementById('background');
```

Comment:
Could this all be pulled out into an e.g. `this.constructCoordinateGridBackground()` ? Just to keep it separated and namespace things a bit.